src/main/java/net/minestom/server/Bootstrap.java

```diff
@@ -20,7 +22,14 @@ public static void bootstrap(String mainClassFullName, String[] args) {
         try {
             ClassLoader classLoader = MinestomRootClassLoader.getInstance();
             startMixin(args);
-            MinestomRootClassLoader.getInstance().addCodeModifier(new MixinCodeModifier());
+            try {
+                MinestomRootClassLoader.getInstance().addCodeModifier(new MixinCodeModifier());
+            } catch (RuntimeException e) {
+                e.printStackTrace();
```

Comment:
Exceptions should be handled using `ExceptionManager` https://github.com/Minestom/Minestom/blob/master/src/main/java/net/minestom/server/exception/ExceptionManager.java#L17

Reply:
No ExceptionManager exists at this point, so it can't be handled through an ExceptionManager